tests/foreman/cli/test_computeresource_azurerm.py

```diff
@@ -67,6 +70,11 @@ def azurerm_hostgroup(
     return hgroup
 
 
+@pytest.fixture(scope='module')
+def module_domain(module_org, module_location):
```

Comment:
Move this to `pytest_fixtures/api_fixtures.py`. So that other test modules can use them.